.revive.toml

```diff
@@ -0,0 +1,25 @@
+ignoreGeneratedHeader = false
```

Comment:
What is this file?

Reply:
Ihor added linter configuration file. Maybe, it's to much?
> What is this file?